docs/Performing-backups.md

```diff
@@ -0,0 +1,103 @@
+These two phases are the following:
+
+- Perform a snapshot on the whole cluster using a default parallelism of 500
+- Perform the upload of the SSTables with a default parallelism of 1.
+
+To speed up the uploads, raise the number of parallel uploads using the `--parallel-uploads` argument.
+
+Once Medusa is setup, you can create a **differential** backup with the following command:
+
+```
+$ medusa backup-cluster --backup-name=<name of the backup>
+```
+
+In order to perform an **full** backup, add the `--mode=full` argument to your command:
+
+```
+$ medusa backup-cluster --backup-name=<name of the backup> --mode=full
+```
+
+`medusa backup-cluster` will invoke `medusa backup` on all nodes in the cluster, as described below.
+
+## Backing up a single node
+
+```
+$ medusa backup --help
+Usage: medusa backup [OPTIONS]
+
+  Backup Cassandra
+
+Options:
+  --backup-name TEXT           Custom name for the backup
+  --stagger INTEGER            Check for staggering initial backups for
+                               duration seconds
+  --mode [full|differential]
+  --help                       Show this message and exit.
+```
+
+Once Medusa is setup, you can create a **differential** backup with the following command:
+
+```
+$ medusa backup --backup-name=<name of the backup>
+```
+
+In order to perform an **full** backup, add the `--mode=full` argument to your command:
+
+```
+$ medusa backup --backup-name=<name of the backup> --mode=full
+```
+
+When executed like this Medusa will:
+
+* Create a snapshot using the Cassandra nodetool command.
+* Upload the snapshot to your configured storage provider.
+* Clear the snapshot from the local node.
+
+Along with the SSTables, Medusa will store three meta files for each backup:
+
+* The complete CQL schema.
+* The token map, a list of nodes and their token ownership.
+* The manifest, a list of backed up files with their md5 hash.
+
+## Full And Differential Backups
+
+All Medusa backups only copy new SSTables from the nodes, reducing the network traffic needed. It then has two ways of managing the files in the backup catalog that we call Full or Differential backups. For Differential backups only references to SSTables are kept by each new backup, so that only a single instance of each SStable exists no matter how many backups it is in. Differential backups are the default and in operations at Spotify reduced the backup size for some clusters by up to 80%.
+
+Full backups create a complete copy of all SSTables on the node each time they run. Files that have not changed since the last backup will be copied in the backup catalog into the new backup (and not copied off the node). In contrast to the differential method which only creates a reference to files. Full backups are useful when you need to take a complete copy and have all the files in a single location.
+
+![Full backups with Medusa for Apache Cassandra](images/medusa_current_full_backups.png)
```

Comment:
Same comment here. Do we want hard links or relative links to the image files? Hard links are good in case we move pages around structurally. Relative links are better if everything moves and stays relative to each other.

Reply:
My preference would go to relative links.

Reply:
:+1: